charts/velero/crds/backups.yaml

```diff
@@ -1,12 +1,16 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: backups.velero.io
   labels:
     app.kubernetes.io/name: velero
   annotations:
     "helm.sh/hook": pre-install
     "helm.sh/hook-delete-policy": "before-hook-creation"
```

Comment:
All the CRDs sync with Velero `v1.4.0` but add the code below to supports helm hook and delete CRDs by the label.
```
  labels:
    app.kubernetes.io/name: velero
  annotations:
    "helm.sh/hook": pre-install
    "helm.sh/hook-delete-policy": "before-hook-creation"
```

Reply:
I was thinking is it possible to generates the labels and annotations by _kubebuilder_ in Velero to make it easier to sync the CRDs to helm-charts by manual or automatic?

Reply:
That's a good question. I have not seen a kubebuilder marker for heml configuration. Let's keep it in mind.

Reply:
Yeah, we want to move over to Kubebuilder for all of our CRDs, but it's going to take some time.

Reply:
Ref to https://github.com/kubernetes-sigs/controller-tools/issues/454